### ISSUE TITLE
Call for Scholarships (CfS)

### ISSUE THREAD

**Issue by TejasAP** [sotm-asia/sotm-asia-2018#25]
Here is the draft for the Scholarship form: https://docs.google.com/forms/d/1L8wOp912JVIP3STbsjEgDIqTSbE4pvPnEGYSWe-8aCY/edit 

Please provide suggestions for questions you would like revised/included.

**Comment by jinalfoflia** [comment]
Thank you @TejasAP for the form! 

Here are a couple of suggestions from my side:

>Would you require a scholarship? 

I'm unsure of having this as a question as this form is primarily meant for one to express interest in scholarship. Would defer to others on this though

**Comment by govvin** [comment]
How about asking whether they need a full, or partial scholarship?

**Comment by TejasAP** [comment]
@govvin The second question addresses your concern. It has been phrased as 'complete' and 'partial' financial support. Would it read better and easier if I replace 'financial support' with 'scholarship'?

**Comment by govvin** [comment]
Yes. I've always thought of it as a 'grant', but financial support work well, too.

**Comment by rasagy** [comment]
Some thoughts:

>Which of the following plans would you prefer:

Unless we explain what these plans are, I don’t think it helps to ask this question. Why not merge this with the next question on participation dependent on scholarship:

**Does your decision to participate depend on the scholarship?**
- [ ] Yes, I can not participate if I don’t get complete support
- [ ] Yes, but I will participate even if I get partial support
- [ ] No, I can participate even if I don’t get any support

This could just be part of the first page.

What do you think?

**Comment by govvin** [comment]
> Which of the following plans would you prefer: *
Only make this question visible if they answer yes in the previous question. Otherwise those who answer no in the previous question is forced to answer this.

**Comment by TejasAP** [comment]
Thanks for the comments, everyone. I have updated the form to reflect the comments better. Please let me know if we need any more changes to be made before the form is made public today.

**Comment by jinalfoflia** [comment]
Here is the draft of the blog post -- https://docs.google.com/document/d/1Sqmu_JqEC6w-hwhMvafCMZ5hEzcO5TPGtx80qo_-Bqk/edit?usp=sharing

Would love to hear your feedback on this.

**Comment by jinalfoflia** [comment]
Thanks for the edits @TejasAP @geohacker, have scheduled the blog post to go out tomorrow morning 10 am, here is the final post - https://medium.com/state-of-the-map-asia/scholarships-362b82c62ac

**Comment by geohacker** [comment]
We closed call for scholarships and are currently reviewing. @jinalfoflia will reach out to each applicant and announce the status.
